cmd/sql.go

```diff
@@ -13,10 +13,12 @@ func NewSQLCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	o := &SQLOptions{IOStreams: ioStreams}
 	cmd := &cobra.Command{
 		Use:   "sql QUERY",
-		Short: "run a SQL query on local dataset(s)",
+		Short: "run a SQL query on local dataset(s) (experimental)",
```

Comment:
Think we need to agree on a consistent "experimental" description. Specifically I saw @dustmop already has a commit in with the following wording: https://github.com/qri-io/qri/commit/c8c98d183f0d46e3f1005ce468dc68f58af021c6#diff-73ef6a20a16eceacdbdf948c7ad07c3eR61

Reply:
very good point. I'll change to match his description.